### PR TITLE
Add accessibility mode attribute to cl-image

### PR DESCRIPTION
**Pull request by strausr**

### CODE REVIEW

File: src/cloudinary-image.component.ts [hunk 1]
```diff
@@ -145,4 +152,16 @@ export class CloudinaryImage
...
   }
+
+  accessibilityModeHandler() {
+    const accessibilityEffect = {
+      'darkmode': {effect: 'tint:75:black'},
+      'brightmode': {effect: 'tint:50:white'},
+      'monochrome': {effect: 'grayscale'},
+      'colorblind': {effect: 'assist_colorblind'}
+    }
+    let transformation = [].concat.apply([], [this.options, accessibilityEffect[this.accessibility]]);
```
Comment (asisayag2) [inline src/cloudinary-image.component.ts]: @strausr , is this the most elegant way of doing this? 
It's ugly!

File: src/cloudinary-image.component.ts [hunk 2]
```diff
@@ -145,4 +152,16 @@ export class CloudinaryImage
...
     this.placeholderComponent.options = placeholderOptions;
   }
+
+  accessibilityModeHandler() {
+    const accessibilityEffect = {
+      'darkmode': {effect: 'tint:75:black'},
+      'brightmode': {effect: 'tint:50:white'},
+      'monochrome': {effect: 'grayscale'},
+      'colorblind': {effect: 'assist_colorblind'}
+    }
```
Comment (asisayag2) [inline src/cloudinary-image.component.ts]: Please take this out of here

File: src/cloudinary-image.component.ts (hunk 1 above)
Comment (strausr) [reply src/cloudinary-image.component.ts]: :/ this is the same convention we used in the placeholder component- https://github.com/cloudinary/cloudinary_angular/blob/master/src/cloudinary-placeholder.component.ts#L42

File: src/cloudinary-image.component.ts (hunk 2 above)
Comment (strausr) [reply src/cloudinary-image.component.ts]: separate file?

Comment (asisayag2) [reply src/cloudinary-image.component.ts]: @strausr, either to a const area in this file or to consts file. 
In the end this should move to the js-core so other FW will be able to use the same pre-defined transformations

File: src/cloudinary-image.component.ts (hunk 1 above)
Comment (asisayag2) [reply src/cloudinary-image.component.ts]: @strausr, not a good reason.
Maybe we can do it more elegantly there as well?

**Pull request merged**